modules/actions/join.js

```diff
@@ -71,6 +71,13 @@ export function actionJoin(ids) {
             graph = actionDeleteWay(way.id)(graph);
         });
 
+        if (rapidInternalFeaturesStorage()('tagnosticRoadCombine')){
```

Comment:
Instead of exporting the raw method for 'featureEnabled' and immediately invoking it, I export an object with the 'featureEnabled' method on it. This changes the invocation of the module to something more akin to what we see elsewhere in the code: 
rapidInternalFeatureStorage().featureEnabled('blah');